astro/src/pages/team/previous.astro

```diff
@@ -0,0 +1,102 @@
+---
+import Layout from '../../layouts/Layout.astro';
+import Section from '../../components/Section.astro';
+---
+
+<Layout title="Our previous contributors">
```

Comment:
Add a breadcrumb for the `/team` link, as Rachael wants. You can see how we do that in the other pages in this directory.